.gitignore

```diff
@@ -6,5 +6,5 @@ Thumbs.db
 /phpunit.xml
 /.idea
 /.vscode
-.phpunit.result.cache
+.phpunit.cache
```

Comment:
While running the test suite locally, I noticed that the PHPUnit cache directory wasn't ignored. I think the naming changed _fairly_ recently, so just reflecting that here.